package.json

```diff
@@ -29,11 +29,16 @@
     "@capacitor/status-bar": "6.0.0",
     "@ionic/angular": "^7.0.0",
     "authentication_frontend_library": "^0.0.57",
+    "chart.js": "^4.4.3",
+    "chartjs-plugin-datalabels": "^2.2.0",
+    "dynamic-form-farhan": "^0.0.6",
```

Comment:
Why are we using 3 packages for dynamic forms?

Reply:
removed the unwanted libraries

Reply:
removed unwanted packages

Reply:
done